db/migrate/20190807183229_add_devise_to_users.rb

```diff
@@ -10,6 +10,7 @@ def change
       ## Recoverable
       t.string   :reset_password_token
       t.datetime :reset_password_sent_at
+      t.boolean  :allow_password_change, default: false, null: false
```

Comment:
You should not modify existing migrations. If you want to add something, add a new one.
Let me know if you don't know why.